deepchem/models/torch_models/layers.py

```diff
@@ -3846,3 +3848,163 @@ def forward(
             P = pair_features
 
         return [A, P]
+
+
+class GlobalMessagePassing(MessagePassing):
+
+    def __init__(self, config):
+        super(GlobalMessagePassing, self).__init__()
+
+        self.dim = config['dim']
+        self.mlp = MultilayerPerceptron(d_input=self.dim,
+                                        d_output=self.dim,
+                                        activation_fn='silu')
+        self.x_edge_mlp = MultilayerPerceptron(d_input=self.dim * 3,
+                                               d_output=self.dim,
+                                               activation_fn='silu')
+        self.linear = nn.Linear(self.dim, self.dim, bias=False)
+
+    def Res(self, m):
```

Comment:
This method reinitializes self.mlp_res, which seems to be a bug for training, please check